.github/workflows/ci.yml

```diff
@@ -9,46 +12,52 @@ jobs:
       run:
         working-directory: docker
 
-    env:
-      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-      DOCKER_REGISTRY: moneymeets/ghd
-
     steps:
       - uses: actions/checkout@v2
 
       - uses: actions/setup-python@v2
         with:
-          python-version: 3.9.5
+          python-version: ${{ env.PYTHON_VERSION }}
 
       - name: Setup Poetry
         uses: moneymeets/action-setup-poetry@master
 
-      - name: Cache Poetry virtualenv
+      - name: Restore virtualenv from Poetry cache
```

Comment:
```suggestion
      - name: Restore Poetry virtualenv from cache
```
ist m.E. korrekter

Reply:
* "Restore _virtualenv_ from _Poetry cache_" = look into _cache belonging to Poetry_ where _virtualenv_ resides and restore it
* "Restore _Poetry virtualenv_ from _cache_" = take _virtualenv owned by Poetry_  and restore it from some _cache_

There is no "more correct" the question is what you want to emphasise. I want to keep same wording in all repositories to simplify the maintenance of the workflows. If you want to change this, implement workflow management in Pulumi first.